README.md

```diff
@@ -124,6 +124,33 @@ invoice = Razorpay::Invoice.create customer_id: customer.id, amount: 100, curren
 
 You can find invoices API documentation at <https://docs.razorpay.com/v1/page/invoices>.
 
+### Plan
+```rb
+# Creating a plan
+plan = Razorpay::Plan.create interval: 1, period: 'monthly', item: { name: 'fake_plan', description: 'fake_desc', currency: 'INR', amount: 500 }, notes: { identifier: 'plan_monthly_super' }
+```
+
+You can find plan API documentation at <https://razorpay.com/docs/subscriptions/api/#plan>.
+
+### Subscription
+```rb
+# Creating a subscription
+subscription = Razorpay::Subscription.create plan_id: plan.id, customer_id: customer.id, start_at: (Time.now + 1.day).to_i, total_count: 3
```

Comment:
1.day .This comes from active_support. Please remove it. Good code.

Reply:
Good catch @tessie . Side-effect of using too much Rails bites back sometimes :stuck_out_tongue: .
I have changed it. Please see if it looks better or any changes needed.